src/actions/saveCoinData.ts

```diff
@@ -66,8 +66,8 @@ const writeFile = async (
   }
 }
 
-const formatCsvFile = async (coin: Partial<CoinMarkets>): Promise<string> => {
-  return await parseAsync(coin, {
+const formatCsvFile = (coin: Partial<CoinMarkets>): string => {
```

Comment:
Likely not an issue but exporting large amount of tickers in one call with a synchronous vs asynchronous parser _might_ have performance issues but I don't know how many people actually use the export feature.


As this function is now synchronous we can remove the `async`/`await` from where we reference this https://github.com/Zidious/crypto-cli/blob/ea4ddfcd8d543d19d4c5a66e41830134dd7a7695/src/actions/saveCoinData.ts#L47-L67